### PR TITLE
Upgrade to be compatible with React Native 0.72.5

### PR DESCRIPTION
**Pull request by michael-adalo**
This change upgrades the Admob component so that it builds on iOS and Android apps built with React Native 0.72.5

Please note that this update is NOT backwards compatible with React Native 0.63.3

### CODE REVIEW

**Comment by michael-adalo** [comment]
[[NOT BACKWARDS COMPATIBLE] Both iOS and Android builds fail to build with after the upgrade for apps with ad-mob banner](https://trello.com/c/vBFRwJYp/515-not-backwards-compatible-both-ios-and-android-builds-fail-to-build-with-after-the-upgrade-for-apps-with-ad-mob-banner)

File: package.json
Comment (enzomonjardin) [inline package.json:26]: Same suggestion as for the other PR, let's block this versions if we can. 

This feedback doesn't block merging the PR though, as it's not a regression (packages weren't locked beforehand)

**Review by enzomonjardin — APPROVED** [review]
LGTM!

Same comment as on the other PR, but I'll leave it up to your criteria to do the change or defer it for the future.

**Pull request merged**
